xds/internal/balancer/ringhash/ringhash.go

```diff
@@ -328,49 +333,37 @@ func (b *ringhashBalancer) ResolverError(err error) {
 //   for some RPCs.
 func (b *ringhashBalancer) UpdateSubConnState(sc balancer.SubConn, state balancer.SubConnState) {
 	s := state.ConnectivityState
-	b.logger.Infof("handle SubConn state change: %p, %v", sc, s)
+	b.logger.Infof("Handle SubConn state change: %p, %v", sc, s)
```

Comment:
Hide this behind a verbosity check?  This seems unnecessarily chatty for most debugging.  (And below)